### ISSUE TITLE
unsolved quests, buildings without number

### ISSUE THREAD

**Issue by Haikuch** [westnordost/StreetComplete#1052]
What about quests that can't be solved in the programmed way? For example there is this house (https://www.openstreetmap.org/way/103360633) that actually does not have a house number. Its also stated so on its homepage http://haus1-berlin.de/ (_Waterloo Ufer **o.Nr.** (Nähe U1/ Hallesches Tor) 10961 Berlin_).

I can of course just remove the quest on my device with "cant say", but because it is not possible to set a number here, it would be great to dismiss the quest completely for streetcomplete. Otherwise, people will stumble about this again and again.

But actually, that would need to set up an extra database, or to put special notes to the object or anything like that. Aside from that, no one knows if there might be a housenumber for this house in the future, but thats a very special case. 

I think beside that example there will always be other cases where it is not possible to solve a problem like its intended by the quest programming, I would appreciate if there would be a more general option than just removing the quest from my personal view.

**Comment by matkoniecz** [comment]
In such cases the standard solution is to make query smarter like it was done to solve #806 #915 #868

**Comment by westnordost** [comment]
Although this issue is written more concise and to the point, it is a duplicate of #351.
Will be implemented, but only after the building quest has been implemented.

**Comment by matkoniecz** [comment]
Query currently excludes buildings with addr:housename, this building has name

https://github.com/westnordost/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/housenumber/AddHousenumber.java#L37

https://www.openstreetmap.org/way/103360633

Maybe 

1) this building should be tagged with addr:housename?

2) buildings with name tags also should be excluded from address quest?

**Comment by westnordost** [comment]
I think `name` is correct.

> buildings with name tags also should be excluded from address quest?

Hmm, in my opinion, not. Names might be i.e. (translated from German)
- Water Police School
- Criminal Court of the State of Hamburg
- Overseas Contor
- Victoria Barracks

All of these have actual house numbers. Especially for the case that there is a certain amenity in a building (only), it will definitely have a name. "House names" as addresses are a thing mostly on the British Isles, as far as I know (don't remember if it is Ireland, Scotland or what)
